tests/metrics/reencryption_firehose.py

```diff
@@ -23,7 +23,7 @@
 
 sys.path.append(os.path.abspath(os.getcwd()))
 
-from umbral.fragments import KFrag
+from umbral.kfrags import KFrag
```

Comment:
Why did you split fragments up? Naming from `kfrags.KFrag` and `cfrags.CapsuleFrag` seems really redundant.

Reply:
It never had sense to group KFrags and CFrags together just because they were "fragments". It also created some unnecessary circular imports, which made difficult to solve type errors found by mypy.

Reply:
Agreed, but I think a better name exists other than `kfrags` and `cfrags`... I'll think about it.

Reply:
Sure, I guess the module name can change if there's a better alternative.